sipa/static/js/agdsn.js

```diff
@@ -94,11 +99,35 @@ function updateStatusWidget(statusMessage, statusEl, tooltipContent) {
     for (const link of statusEl.getElementsByTagName("a")) {
         link.classList.remove("placeholder");
         link.innerHTML = statusMessage[get_language()];
+        // tooltip
+        if (tooltipContent) {
+            link.dataset.bsTitle = tooltipContent;
+            new bootstrap.Tooltip(link);
+        }
     }
-    // tooltip
+}
+
+/**
+ * @param {HTMLElement} statusEl
+ * @param {?string} tooltipContent
+ */
+function updateMaintenancesPartOfStatusWidget(statusEl, tooltipContent) {
     if (tooltipContent) {
-        statusEl.dataset.bsTitle = tooltipContent;
-        new bootstrap.Tooltip(statusEl);
+        for (const link of statusEl.getElementsByTagName("a")) {
+            const parent = link.parentElement;
+            if (parent.tagName.toLowerCase() == "div") {
+                parent.innerHTML += `<br><span class="bi-clock-fill text-muted"></span>`
```

Comment:
Ich denke `bi-clock` passt besser; die graue Uhr wirkt so fett und da wirkt das auf mich etwas unbalanciert. Wie siehst du das?

![image](https://github.com/agdsn/sipa/assets/10060406/3754c126-c505-46e7-acd0-789856e1aca7)

![image](https://github.com/agdsn/sipa/assets/10060406/5cce8842-e266-4811-8abc-18c0d1b781e1)

```suggestion
parent.innerHTML += `<br><span class="bi-clock text-muted"></span>`
```

Reply:
Ohne Widerrede angenommen :)